spec/views/commodities/_commodity.html.erb_spec.rb

```diff
@@ -6,61 +6,78 @@
     rendered
   end
 
-  describe 'for commodity' do
-    context 'with children' do
-      let(:heading) { build :heading, commodities: [parent, child] }
+  let(:uk_service) { true }
+  let(:commodity) { build :commodity }
 
-      let(:parent) do
-        attributes_for :commodity, producline_suffix: producline_suffix,
-                                   number_indents: 2
-      end
+  before do
+    allow(TradeTariffFrontend::ServiceChooser).to receive(:uk?).and_return(uk_service)
```

Comment:
Minor but there's an existing shared_context to mock out the service? `include_context 'with XI service'` ?

https://github.com/trade-tariff/trade-tariff-frontend/blob/main/spec/views/rules_of_origin/_rules_table.html.erb_spec.rb#L42